src/sbvr-api/odata-response.ts

```diff
@@ -1,7 +1,7 @@
-declare module '@resin/abstract-sql-compiler' {
+declare module '@balena/abstract-sql-compiler' {
 	interface AbstractSqlTable {
 		fetchProcessingFields?: {
-			[field: string]: (field: any) => Bluebird<any>;
+			[field: string]: NonNullable<typeof sbvrTypes[string]['fetchProcessing']>;
```

Comment:
I might be wrong, but I think that this isn't `NonNullable`
See: https://github.com/balena-io-modules/sbvr-types/blob/07981e2ed3fe40d638175dd97de71e3d1474e16e/src/types/date.ts#L12-L17

Reply:
It is in this case because this property is built up as
```
const getFetchProcessingFields = (table: AbstractSqlTable) => {
	if (table.fetchProcessingFields == null) {
		table.fetchProcessingFields = _(table.fields)
			.filter(
				({ dataType }) =>
					sbvrTypes[dataType] != null &&
					sbvrTypes[dataType].fetchProcessing != null,
			)
			.map(({ fieldName, dataType }) => {
				const odataName = sqlNameToODataName(fieldName);
				return [odataName, sbvrTypes[dataType].fetchProcessing];
			})
			.fromPairs()
			.value();
	}
	return table.fetchProcessingFields!;
};
```
so it filters out any that are null

Reply:
Scratch that, I thought I saw a `ReturnType` in there...